### PR TITLE
Add quiz category page

### PR DESCRIPTION
**Pull request by sah-shah7**
-Added Categories Page
-Modified the code to tailor towards the purpose of the page
image

### CODE REVIEW

File: stylesheets/quiz-category.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    padding: 0 1rem 0 1rem;
+}
+
+.vertical-card-image{
+    max-inline-size: 100%;
+    block-size: auto;
+    object-fit: cover;
+    font-size: inherit;
+    display: inline-block;
+    border-radius:8px;
```
Comment (sharath-io) [inline stylesheets/quiz-category.css:22]: You can use rem for this as well. This is just suggestion not compulsory.

**Review by sharath-io — APPROVED** [review]
Everything looks good. Using rem is suggestion not compulsory.

**Comment by sah-shah7** [comment]
PR approved. Merging

File: src/pages/category/quiz-category.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+                <div class="column-flex">
+                    <div class="main-container m-y-auto m-x-b-auto m-x-t text-center">
+                        <h1 class="heading-black text-capitalize">Quiz Categories</h1>
+                        <p class="lead pd-x-1">Choose Wisely!</p>
+                    </div>
+
+                    <div class="card-category-container row-flex row-flex-content m-y-auto m-x-b-auto m-x-t">
+
+
```
Comment (zabihshaik) [inline src/pages/category/quiz-category.html:53]: Better to remove extra spaces.

File: src/pages/category/quiz-category.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                            </p>
+                        </div>
+
+                        <div class="card-buttons">
+                            <span class="btn-add-to-card">
+                                <a href="./src/pages/category/quiz-category.html"><button class="btn-1 quiz-button-primary">Play</button></a>
+                            </span>
+                        </div>
+                    </div>
+
```
Comment (zabihshaik) [inline src/pages/category/quiz-category.html:90]: Here as well. Better to use Prettier.

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just a couple of comments.

**Pull request merged**